### PR TITLE
Update NextVersion.md

### PR DESCRIPTION
**Pull request by GerardasB**
## Changes

Add missing notes to `NextVersion.md`

## Testing

N/A

### CODE REVIEW

**Review by saskliutas — APPROVED** [review]

File: docs/changehistory/NextVersion.md [hunk 1]
```diff
@@ -7,11 +7,31 @@ publish: false
...
+    - [`IconWebComponent`](#iconwebcomponent)
+    - [Fixed `ConditionalIconItem` functionality](#fixed-conditionaliconitem-functionality)
   - [@itwin/components-react](#itwincomponents-react)
 
+## @itwin/appui-react
+
+### Popout widgets
+
+- Resolved an issue that prevented opening popouts from within an iFrame. Please note that sandboxed iFrames still require the allow-popups attribute.
+- Eliminated the logic responsible for storing the popped-out widget's location on the screen. Now, when the user closes the widget, it will reappear in the same position it was last closed.
```
Comment (AntanasBuk) [inline docs/changehistory/NextVersion.md:25]: If you removed the logic, doesn't that mean that the widget **WON'T** reappear in the same location?

File: docs/changehistory/NextVersion.md [hunk 2]
```diff
@@ -7,11 +7,31 @@ publish: false
...
+
+### Badge icons
+
+An issue with the conversion of toolbar item definitions has been fixed. This fix addresses the problem where badge icons were not rendering correctly.
```
Comment (AntanasBuk) [inline docs/changehistory/NextVersion.md:29]: We previously added fixes to a separate "Fixes" header.

**Review by AntanasBuk — CHANGES_REQUESTED** [review]

File: docs/changehistory/NextVersion.md (hunk 1 above)
Comment (GerardasB) [reply docs/changehistory/NextVersion.md]: Good catch - reworded.

File: docs/changehistory/NextVersion.md (hunk 2 above)
Comment (GerardasB) [reply docs/changehistory/NextVersion.md]: That's a good point, pushed a fix to separate headings based on change type instead of component/feature.

**Pull request merged**